### PR TITLE
Select target action does not include the line number when used from the editor

### LINKED ISSUE #24: Select target action does not include the line number when used from the editor
The basic feature 
    ```Select File in Project view, Alt-F1 from editor, invoke "Open in Git Host" ```
works correctly.

But
    ```Open file in editor, place cursor, Alt-F1 from editor, invoke "Open in Git Host"``` -> 
ACTUAL
Browser is opened but without correct line number
EXPECTED
Browser is opened with correct line number

### PR DESCRIPTION
**Pull request by markiewb**
Fixes #24. 
Please review and merge!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,8 @@ Make sure you have registered your projects root under the version control prefe
 
 Open a project file that is under Git version control in the editor:
 
-      View → Open in Git host
+      View → Open in Git host or
+      Alt-F1 → Open in Git host
```
Comment (ben-gibson) [inline README.md]: I'm a little reluctant to include shortcuts in the README as they aren't always consistent between different OS.

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
   <change-notes><![CDATA[
     <ul>
       <li>
-          Ability to open a commit from the VCS log tool window <a href="https://github.com/ben-gibson/jetbrains-open-in-git-host/issues/22">#22</a>.
+          Fixed: "Open in... Git Host" action works also from editor (It uses the line number from the current editor) <a href="https://github.com/ben-gibson/jetbrains-open-in-git-host/issues/24">#24</a> (PR by markiewb)
```
Comment (ben-gibson) [inline resources/META-INF/plugin.xml]: Could this be updated to something like Fixed: Select target action now uses the correct line number when used from the editor.

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,3 +14,7 @@ All Notable changes to `Open in Git host` will be documented in this file
 ## 1.5.6 - 2016-09-26
 
 - Ability to open a specific commit from the VCS log tool window.
+
+## 1.5.7 - 2016-12-04
+
+-  Fixed: "Open in... Git Host" action works also from editor (It uses the line number from the current editor) (#24 PR by markiewb)
```
Comment (ben-gibson) [inline CHANGELOG.md]: Could this be updated to something like Fixed: Select target action now uses the correct line number when used from the editor.

**Comment by markiewb** [comment]
@ben-gibson: I updated the descriptions based on your recommendations. Please review!

**Pull request merged**